libraries/AP_HAL_ChibiOS/hwdef/common/malloc.c

```diff
@@ -530,3 +530,47 @@ void* get_addr_mem_region_end_addr(void *addr)
     }
     return 0;
 }
+
+#if defined(SECURE) && SECURE
```

Comment:
can probably use normal realloc

Reply:
had to move std_realloc to malloc.c for this.